src/test/java/org/apache/lucene/store/TestSmbSimpleFSDirectory.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.lucene.store;
+
+import java.io.File;
+import java.io.IOException;
+
+public class TestSmbSimpleFSDirectory extends BaseDirectoryTestCase {
```

Comment:
So we need to change this here as well: https://github.com/elasticsearch/elasticsearch-parent/blob/master/pom.xml#L519
And in 1.x branch: https://github.com/elasticsearch/elasticsearch-parent/blob/1.x/pom.xml#L519

Otherwise, if/when we will remove explicit settings for `junit4-maven-plugin` plugin those tests won't be run again I guess.